tests/commands/test_set.py

```diff
@@ -339,81 +339,63 @@ def test_set_dryrun_flag(repo: Repo, set_values: list[str]) -> None:
     repo.fsck()
 
 
-@pytest.mark.repo_files("laptop_macbook_pro.0",
-                        "dir1/laptop_macbook_pro.1",
-                        "dir1/dir2/laptop_macbook_pro.2",
-                        "dir1/dir2/dir3/laptop_macbook_pro.3",
-                        "dir1/dir2/dir3/dir4/laptop_macbook_pro.4",
-                        "dir1/dir2/dir3/dir4/dir5/laptop_macbook_pro.5",
-                        "dir1/dir2/dir3/dir4/dir5/dir6/laptop_macbook_pro.6",)
+@pytest.mark.repo_files(
+    "laptop_macbook_pro.0",
+    "dir1/laptop_macbook_pro.1",
+    "dir1/dir2/laptop_macbook_pro.2",
+    "dir1/dir2/dir3/laptop_macbook_pro.3",
+    "dir1/dir2/dir3/dir4/laptop_macbook_pro.4",
+    "dir1/dir2/dir3/dir4/dir5/laptop_macbook_pro.5",
+    "dir1/dir2/dir3/dir4/dir5/dir6/laptop_macbook_pro.6",)
 @pytest.mark.parametrize('set_values', values)
-def test_set_depth_flag(repo: Repo, set_values: list[str]) -> None:
+@pytest.mark.parametrize('depth,expected', [
```

Comment:
Yes that is intentional (and I use it all over the `onyo get` tests too). Is this problematic?

Reply:
I have not done so before. What is the behavior? My imagination assumes it results in X * Y runs.

Reply:
Yes, it takes each combination of `set_values` and `depth,expected`. The reason to put `depth,expected` together is because the expected output depends on the given depth value, so mixing them up is a bad idea. The `set_values` part switches up the keys that are used to call `onyo set` and is from the original test that I saw no need to remove. 

The functionality remains the same to how it was before (testing all combinations of `set_values` and `depth`), except now `depth` is parametrized instead of hard-coded in the test.

Reply:
Cool; thanks for explaining. :-)